ports/stm/boards/weact-h723vg-lcd/mpconfigboard.mk

```diff
@@ -0,0 +1,64 @@
+USB_VID = 0x0483
+USB_PID = 0x5740
```

Comment:
This is a generic STM VID/PID. If you don't work for WeAct you can request a USB PID from: https://pid.codes/ If you do work for WeAct, then you should sublicense a unique PID from STM or acquire your own VID from the USB folks.